test/Maestro/US/gratitudeGame_2Players.yml

```
@@ -18,6 +18,10 @@ name: Playing the gratitude game with 2 players without AI
       id: "Hero6.svg"
     direction: "DOWN"
 - tapOn: "See the rules"
+- tapOn: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)